pages/common/for.md

```diff
@@ -1,7 +1,7 @@
 # for
 
 > Shell loop over parameters.
-> More information: <https://man.archlinux.org/man/for.n>.
+> More information: <https://manned.org/for>.
```

Comment:
Hmm, this link (as well as the man.archlinux.org one) goes to the Tcl documentation for `For`, not the shell builting 🤔

Reply:
Good catch. I suppose we should include the correct section in the link? https://manned.org/for.1

Reply:
Yes, that's a good one. Although I've always hated that man page because it's the same for all shell builtins and is not informative at all.

Reply:
What if we went with https://pubs.opengroup.org/onlinepubs/9699919799//utilities/V3_chap02.html#tag_18_09_04_03 then?

Reply:
Sure, that seems fine.  It's a little long, but it will do.